spec/features/idv/doc_capture/document_capture_step_spec.rb

```diff
@@ -131,7 +131,7 @@
 
     it 'proceeds to the next page with valid info and logs analytics info' do
       allow(Analytics).to receive(:new).and_return(fake_analytics)
-      expect(Analytics).to receive(:new).with(hash_including(user: user))
+      expect(Analytics).to receive(:new).with(hash_including(user: AnonymousUser))
```

Comment:
This assertion isn't particularly useful for us. Can we update the one below include the expected `user_id` in the hash?

```
      expect(fake_analytics).to have_logged_event(
        Analytics::DOC_AUTH + ' submitted',
        step: 'document_capture',
        result: 'Passed',
        billed: true,
        user_id: user.uuid,
      )
```

Reply:
still making updates ...

Reply:
waiting to see what specs are broken on this last change...